### PR TITLE
Fix MultiPolygon validation

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request fix the multipolygon validation (and the associated test).
Tha data used for testing were false.

### CODE REVIEW

File: geojson/validation.py [hunk 1]
```diff
@@ -58,14 +58,19 @@ def is_valid(obj):
                           'must be equivalent')
 
     if isinstance(obj, geojson.MultiPolygon):
-        if checkListOfObjects(obj['coordinates'],
-                              lambda x: len(x) >= 4 and x[0] == x[-1]):
+        if checkListOfObjects(obj['coordinates'], lambda x: is_polygon(x)):
```
Comment (frewsxcv) [inline geojson/validation.py]: Can you wrap this line? It's one character too long for flake8 :P 

https://travis-ci.org/frewsxcv/python-geojson/jobs/84943678

Comment (noirbizarre) [inline geojson/validation.py]: It wasn't this line, but I fixed it in the last commit:
https://travis-ci.org/frewsxcv/python-geojson/jobs/84946771

**Comment by frewsxcv** [comment]
Reopened to restart travis

File: geojson/validation.py [hunk 2]
```diff
@@ -58,14 +58,19 @@ def is_valid(obj):
...
             return output('the "coordinates" member must be an array'
                           'of Polygon coordinate arrays')
 
     return output('')
 
 
+def is_polygon(coords):
+    lengths = all([len(elem) >= 4 for elem in coords])
+    isring = all([elem[0] == elem[-1] for elem in coords])
```
Comment (frewsxcv) [inline geojson/validation.py]: You can remove the inner brackets in these so they become generators

```python
    lengths = all(len(elem) >= 4 for elem in coords)
    isring = all(elem[0] == elem[-1] for elem in coords)
```

**Comment by noirbizarre** [comment]
Done.

BTW, I'm really interested in improving validation with more detailled feedback.
I can submit a pull-request but I just wan't to know what constraint do you have.
I explain myself: in the case of a multipolygon, it's possible to have multiple validation error but the current mecanism only allow one feedback message.
To do so, I need to transform ``validation['message']`` into an array if it's OK.
If not, I will try to agregate messages into a single one

**Comment by frewsxcv** [comment]
> but I just wan't to know what constraint do you have.

I have no constraints with the validation format. Do whatever you think makes the most sense to you

> To do so, I need to transform validation['message'] into an array if it's OK.

Sounds goods to me! I would just rename the key to `'messages'` and then just make sure it is always an array. I'll make sure to publish a new major version following that change.

In the meantime, thanks for your contribution!

**Comment by frewsxcv** [comment]
Also, feel free to add yourself to the credits in the README for your daily dose of egotism

**Comment by noirbizarre** [comment]
Thanks !
I will in the next coming pull-request ;)

**Comment by frewsxcv** [comment]
Version 1.3.1 has been released which includes this fix. Thanks again!

https://pypi.python.org/pypi/geojson/1.3.1

**Pull request merged**